antenna/throttler.py

```diff
@@ -72,10 +82,16 @@ class Throttler(RequiredConfigMixin):
     required_config = ConfigOptions()
     required_config.add_option(
         'throttle_rules',
-        default='antenna.throttler.mozilla_rules',
+        default='antenna.throttler.MOZILLA_RULES',
         doc='Python dotted path to ruleset',
         parser=parse_attribute
     )
+    required_config.add_option(
+        'products',
+        default='antenna.throttler.MOZILLA_PRODUCTS',
+        doc='Python dotted path to list of supported products',
+        parser=parse_attribute
+    )
```

Comment:
I made this configurable because I secretly hope there are other people using Antenna out there.